### PR TITLE
Linux - Changing File Ownership With `chown`

### PR DESCRIPTION
**Pull request by kapnobatai137**
Problem mentioned in #2164

- new practice question
- new notes
- specified that `chown` can only be used with root access

TODO:
- [ ] reply & resolve comment in this insight

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md?ref=e41cb86627d9410ebc41e9c76becebd548199921)

File: linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md [hunk 1]
```diff
@@ -1,98 +1,81 @@
...
-
 ---
 
 # Changing File Ownership With `chown`
 
 ---
 ## Content
 
-The `chown` command allows us to change the both the user and group ownership of files and directories.  The user running this command must have write permission to the file in question.
+The `chown` command allows us to change the both the user and group ownership of files and directories. The user running this command must have root access.
```
Comment (Loopiezlol) [inline linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md:23]: I think you can run chwon without root access if you're the file owner already

File: linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md [hunk 2]
```diff
@@ -1,98 +1,81 @@
...
 
 To change the user who owns a specific file run the following (`jesse` is just an example username):
 
 ```shell
-$ chown jesse /path/to/some/file
+sudo chown jesse /path/to/some/file
```
Comment (Loopiezlol) [inline linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md:28]: sudo might not be needed

File: linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md (hunk 1 above)
Comment (Loopiezlol) [reply linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md:23]: it's a bit weird to read that at the end

File: linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md [hunk 3]
```diff
@@ -1,98 +1,81 @@
...
-
-???
+To change the owner of a file using `chown` you need to ???.
 
-* You have write permissions for the file.
-* You have execute permissions for the file.
-* You have read permissions for the file.
-* The file has the sticky bit permission.
+* have root access
+* be the owner of the file
```
Comment (Loopiezlol) [inline linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md:63]: this is misleading, 2nd answer is also correct

File: linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md [hunk 4]
```diff
@@ -1,98 +1,81 @@
...
 
 To change only the group run:
 
 ```shell
-$ chown :new-group /path/to/some/file
+sudo chown :new-group /path/to/some/file
 ```
 
-If you don't have permission to change the owner of a file but you have permission to use `sudo` then you can run the `chown` command using `sudo`, e.g., `sudo chown jesse:new-user /path/to/some/file`.
+> 💡 If you don't have root access, you can change the permissions of the file using `chmod` **if you are the owner of that file**. You can't change ownership (with `chown` or `chgrp`) without root access.
```
Comment (Loopiezlol) [inline linux/user-and-file-management/user-permissions/changing-file-ownership-with-chown.md:43]: maybe explain what chmod is doing in comparison with chown

**Pull request merged**